

- remove unused local

- whitespace

- put_bh() is faster than brelse()

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/ext3/inode.c |   11 ++++-------
 1 files changed, 4 insertions(+), 7 deletions(-)

diff -puN fs/ext3/inode.c~ext3_bread-cleanup fs/ext3/inode.c
--- 25/fs/ext3/inode.c~ext3_bread-cleanup	2004-06-27 17:24:10.239331816 -0700
+++ 25-akpm/fs/ext3/inode.c	2004-06-27 17:24:10.245330904 -0700
@@ -911,20 +911,17 @@ struct buffer_head *ext3_bread(handle_t 
 			       int block, int create, int *err)
 {
 	struct buffer_head * bh;
-	int prev_blocks;
 
-	prev_blocks = inode->i_blocks;
-
-	bh = ext3_getblk (handle, inode, block, create, err);
+	bh = ext3_getblk(handle, inode, block, create, err);
 	if (!bh)
 		return bh;
 	if (buffer_uptodate(bh))
 		return bh;
-	ll_rw_block (READ, 1, &bh);
-	wait_on_buffer (bh);
+	ll_rw_block(READ, 1, &bh);
+	wait_on_buffer(bh);
 	if (buffer_uptodate(bh))
 		return bh;
-	brelse (bh);
+	put_bh(bh);
 	*err = -EIO;
 	return NULL;
 }
_
